.github/workflows/_ci.yaml

```diff
@@ -55,7 +54,7 @@ jobs:
       BUILD_DATE: ${{ needs.metadata.outputs.BUILD_DATE }}
       BASE_IMAGE: ${{ needs.build-jax.outputs.DOCKER_TAG_MEALKIT }}
       CONTAINER_NAME: upstream-t5x
-      DOCKERFILE: .github/container/Dockerfile.t5x
+      DOCKERFILE: .github/container/Dockerfile.pax.${{ inputs.ARCHITECTURE }}
```

Comment:
This says pax instead of t5x

Reply:
Indeed, that and another line in `nightly-t5x-build.yaml` was copy-paste nonsense. Thanks!